squidb/src/com/yahoo/squidb/sql/SqlUtils.java

```diff
@@ -7,7 +7,6 @@
 
 import android.text.TextUtils;
 
-import java.math.BigDecimal;
```

Comment:
Is this no longer supported? Or do we just no longer need to check for it specifically?